app/src/screens/screens.tsx

```diff
@@ -0,0 +1,15 @@
+import about from './about';
+import changelog from './changelog';
+import preferences from './preferences';
+import shortcuts from './shortcuts';
+import task from './task';
+
+export const SCREENS = {
+  about,
+  changelog,
+  preferences,
+  shortcuts,
+  task,
+};
+
+export type Screens = keyof typeof SCREENS
```

Comment:
Can you name this `IScreeens`, I want to follow the `IType` convention.

Reply:
This makes sense. I'll start using that on my own projects too.